bufferserver/src/main/java/com/datatorrent/bufferserver/server/Server.java

```diff
@@ -73,9 +73,12 @@
   private InetSocketAddress address;
   private final ExecutorService serverHelperExecutor;
   private final ExecutorService storageHelperExecutor;
+  private boolean backPressureEnabled;
 
   private byte[] authToken;
 
+  private static final boolean BACK_PRESSURE_ENABLED = !Boolean.getBoolean("BUFFERSERVER.BACK_PRESSURE_DISABLE");
```

Comment:
It will be much easier to remove it later if it is final static of the DataList itself.

True for netlet, AFAIK all apex are long one.